command/daemon/command.go

```diff
@@ -128,9 +128,11 @@ func (c *command) Execute(cmd *cobra.Command, args []string) {
 		go newServer.Boot()
 	}
 
-	// Listen to OS signals.
+	// Listen to OS signals. Pressing Ctrl+C produces SIGINT. SIGTERM handled
+	// here for graceful HTTP server shutdown and return of successful exit
+	// status.
```

Comment:
Since we just talked about it, can we add more docs and explain why that is and maybe also link to the k8s docs and some Docker background info if necessary?